### PR TITLE
Add a more real Java testcase

### PR DESCRIPTION
**Pull request by cmoulliard**
See Issue #74

### CODE REVIEW

File: spring-cloud-kubernetes-config/src/test/java/org/springframework/cloud/kubernetes/config/ConfigMapsSpringBootTest.java [hunk 1]
```diff
@@ -75,11 +76,13 @@ public void beforeTest() {
 			.once();
 	}
 
+	@Ignore
```
Comment (iocanel) [inline spring-cloud-kubernetes-config/src/test/java/org/springframework/cloud/kubernetes/config/ConfigMapsSpringBootTest.java]: The ignored test cases, are actually pretty valid for the core module, which is responsible for injecting the client. I would move them there (unignored ofc).

File: spring-cloud-kubernetes-config/pom.xml [hunk 2]
```diff
@@ -76,6 +76,12 @@
 			<artifactId>spock-spring</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.rest-assured</groupId>
+			<artifactId>rest-assured</artifactId>
+			<version>3.0.2</version>
```
Comment (iocanel) [inline spring-cloud-kubernetes-config/pom.xml]: Can we use a placeholder for that?

Comment (cmoulliard) [inline spring-cloud-kubernetes-config/pom.xml]: > placeholder

Do you mean "add this dependency within a parent pom file" ?

File: spring-cloud-kubernetes-config/src/test/java/org/springframework/cloud/kubernetes/config/ConfigMapsSpringBootTest.java (hunk 1 above)
Comment (cmoulliard) [inline spring-cloud-kubernetes-config/src/test/java/org/springframework/cloud/kubernetes/config/ConfigMapsSpringBootTest.java]: Do you want that we move this test 

```
	@Test
	public void testConfig() {
		assertEquals(config.getMasterUrl(),mockClient.getConfiguration().getMasterUrl());
		assertEquals(config.getNamespace(),mockClient.getNamespace());
	}
```
within another a new Test Class of the spring-cloud-kubernetes-core project ?

File: spring-cloud-kubernetes-config/pom.xml (hunk 2 above)
Comment (iocanel) [inline spring-cloud-kubernetes-config/pom.xml]: I mean adding a property like 

    <restassured.version>3.0.2</restassured.version>

either on the top of the current pom, or maybe in the parent (if we are using restassured in more than one places).

File: spring-cloud-kubernetes-config/src/test/java/org/springframework/cloud/kubernetes/config/ConfigMapsSpringBootTest.java (hunk 1 above)
Comment (iocanel) [inline spring-cloud-kubernetes-config/src/test/java/org/springframework/cloud/kubernetes/config/ConfigMapsSpringBootTest.java]: Short answer: Yes

Longer answer: Having an ignored test, that doesn't test anything specific to the current module is not really clean. So I was thinking that we could move it to the core where its more relevant.

**Comment by cmoulliard** [comment]
Squashed into one commit. There are no longer @Ignore annotations with the latest commit. Do you want to do something else ?

**Pull request merged**